definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -54,5 +54,66 @@
     "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/petition-updated",
     "RetriesTimeoutURLSubmittedEvent": "120,120",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
It doesn't matter for CCD, but it's quite useful to use current date here.

Reply:
Updated